test/cases/expressions/xhp-enum.hack

```diff
@@ -1,3 +1,3 @@
-class :a {
+xhp class a {
   attribute enum {'a', 'b', 1} denum = 1 @required;
 }
```

Comment:
This class is more about testing the `attribute enum` feature for xhp-v3, so I'd rather we don't update this file (and the equivalent xhp-enum.exp file). The xhp-class-v4.hack test can stand on it's own as a test for the v4 syntax test.